### PR TITLE
revert changes to follow_path.py from request 4350

### PR DESCRIPTION
**Pull request by ckrmml**
## Short Description:
request 4350 changes to follow_path.py crash bot
## Fixes/Resolves/Closes (please use correct syntax):
- reverted the changes

### CODE REVIEW

File: pokemongo_bot/cell_workers/follow_path.py [hunk 1]
```diff
@@ -139,8 +140,8 @@ def work(self):
             'position_update',
             formatted="Walking from {last_position} to {current_position}, distance left: ({distance} {distance_unit}) ..",
             data={
-                'last_position': (last_lat, last_lng, last_alt),
-                'current_position': (lat, lng, alt),
+                'last_position': (last_lat, last_lng, 0),
+                'current_position': (lat, lng, 0),
```
Comment (solderzzc) [inline pokemongo_bot/cell_workers/follow_path.py]: 0 is not good. we have random altitude for now, please help to check if sending 0 to server.

Comment (ckrmml) [inline pokemongo_bot/cell_workers/follow_path.py]: pr 4350 crashes the bot and this is just copied over from the commit before, so better this than a crash i assume

**Comment by ckrmml** [comment]
no experience in python but the last version here is the one i run atm and it works. sending zero 0s to the server and showing altitude in log

File: pokemongo_bot/cell_workers/follow_path.py [hunk 2]
```diff
@@ -99,11 +101,14 @@ def find_closest_point_idx(self, points):
...
     def work(self):
-        last_lat, last_lng, last_alt = self.bot.api.get_position()
+        last_lat = self.bot.api._position_lat
+        last_lng = self.bot.api._position_lng
+        last_alt = self.bot.api._position_alt
 
         point = self.points[self.ptr]
         lat = float(point['lat'])
         lng = float(point['lng'])
+        alt = float(point['alt'])
```
Comment (kanemasa1987) [inline pokemongo_bot/cell_workers/follow_path.py]: json files do not contain alt.

Comment (ckrmml) [inline pokemongo_bot/cell_workers/follow_path.py]: gpx files do, and this is the change that allows the bot to follow path provided by gpx after your pr forced it to crash

**Comment by kanemasa1987** [comment]
Easiest fix would probably put

_, _, alt = self.bot.api.get_position()

right before emit.

File: pokemongo_bot/cell_workers/follow_path.py (hunk 2 above)
Comment (ckrmml) [inline pokemongo_bot/cell_workers/follow_path.py]: and gpsies.com json do contain elevation
```
{"data":{"trackData":[[{"lon":-118.48879337310791,"lat":34.031891963311,"ele":65},{"lon":-118.49720478057863,"lat":34.0323898728239,"ele":69}...
```

Comment (kanemasa1987) [inline pokemongo_bot/cell_workers/follow_path.py]: There is an option to read json files, around line 41, and they don't read alt.

Comment (kanemasa1987) [inline pokemongo_bot/cell_workers/follow_path.py]: if 'alt' in point:
    alt = float(point['alt'])
else:
    alt = uniform(self.bot.config.alt_min, self.bot.config.alt_max)

should keep everything good

**Pull request merged**